bowling/inc/bowlingGameStub.hpp

```diff
@@ -0,0 +1,109 @@
+#pragma once
```

Comment:
isn't it mainly a copy of BowlingGame class? I can see a lot of code duplication.

Reply:
Removed redundant Stub.